tests/regression/03-practical/18-no_ctx_box.c

```diff
@@ -1,4 +1,4 @@
-// SKIP PARAM: --no mutex --set solver "'new'" --no-context base --no-context escape
+// SKIP PARAM: --set ana.activated[-] mutex --set solver "'new'" --set ana.ctx_insens[+] base --set ana.ctx_insens[+] escape --set ana.base.privatization none
```

Comment:
I was about to say that we don't have a "new" solver, but apparently we do...
https://github.com/goblint/analyzer/blob/b899e74ce22a5631bc811bc06f901256511eabe0/src/solvers/sLR.ml#L536-L537